catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -348,7 +359,8 @@ public void error(SAXParseException exception) throws SAXException {
     @Override
     public void fatalError(SAXParseException exception) throws SAXException {
         String message = "Fatal Error: " + getParseExceptionInfo(exception);
```

Comment:
Ah yes. Intellij caught them as code smells - concats inside appends - and I had it fix them for me. I could certainly replace that concat as well. Give me a sec.